common/lib/xmodule/xmodule/studio_editable.py

```diff
@@ -1,15 +1,21 @@
 """
 Mixin to support editing in Studio.
 """
+from xblock.core import XBlock
 from xmodule.x_module import AUTHOR_VIEW, STUDENT_VIEW, module_attr
 
 
+@XBlock.needs('mako')
 class StudioEditableBlock:
     """
     Helper methods for supporting Studio editing of XBlocks.
 
     This class is only intended to be used with an XBlock!
     """
+    _services_requested = {
+        'mako': 'need',
```

Comment:
`StudioEditableBlock` needs the `mako` service now, but since it's not an XBlock, it doesn't have the `_services_requested` class property itself, so I borrowed this trick from [AuthoringMixin](https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/cms/lib/xblock/authoring_mixin.py#L22-L24) to avoid this error incurred while initializing the decorated class:

```
AttributeError: type object 'StudioEditableBlock' has no attribute '_services_requested'
```

However, it can be an empty dict and perform the same duty.. I'll fix that.

Reply:
Added comment about this: https://github.com/edx/edx-platform/pull/28571/files#diff-37de9c229ad553ea6e2b8eb6b8551bd17dab0f41057d053a41c3f1dd6a791becR15-R17